spec/requests/api/admin/create_spec.rb

```diff
@@ -0,0 +1,100 @@
+RSpec.describe 'POST /api/admin/articles', type: :request do
+
+  describe 'Successfully with valid params and user' do
+    let(:journalist) { create(:user, role: 'journalist')}
```

Comment:
IF you decide to follow my advice and add a second factory, this can be refactored to:

```ruby 
let(:journalist) { create(:journalist) }
```

Reply:
ok

Reply:
please correct this